src/main/resources/assets/immersiveweapons/lang/zh_cn

```diff
@@ -0,0 +1,687 @@
+	"block.immersiveweapons.dying_soldier_head": "濒死的苏联士兵头颅",
+	"block.immersiveweapons.wandering_warrior_head": "流浪战士头颅",
+	"block.immersiveweapons.hans_head": "全能的汉斯头颅",
+	"entity.immersiveweapons.dying_soldier": "濒死的苏联士兵",
+	"entity.immersiveweapons.minuteman": "美国民兵",
+	"entity.immersiveweapons.field_medic": "战地医疗兵",
+	"entity.immersiveweapons.wandering_warrior": "流浪战士",
+	"entity.immersiveweapons.hans": "全能的汉斯",
+	"entity.immersiveweapons.wooden_arrow": "木箭",
+	"entity.immersiveweapons.stone_arrow": "石箭",
+	"entity.immersiveweapons.golden_arrow": "金箭",
+	"entity.immersiveweapons.copper_arrow": "铜箭",
+	"entity.immersiveweapons.iron_arrow": "铁剑",
+	"entity.immersiveweapons.cobalt_arrow": "钴箭",
+	"entity.immersiveweapons.diamond_arrow": "钻石箭",
+	"entity.immersiveweapons.netherite_arrow": "下界合金箭",
+	"entity.immersiveweapons.wooden_musket_ball": "木制弹丸",
+	"entity.immersiveweapons.stone_musket_ball": "石制弹丸",
+	"entity.immersiveweapons.golden_musket_ball": "金制弹丸",
+	"entity.immersiveweapons.copper_musket_ball": "铜制弹丸",
+	"entity.immersiveweapons.iron_musket_ball": "铁制弹丸",
+	"entity.immersiveweapons.cobalt_musket_ball": "钴制弹丸",
+	"entity.immersiveweapons.diamond_musket_ball": "钻石制弹丸",
+	"entity.immersiveweapons.netherite_musket_ball": "下界合金制弹丸",
+	"entity.immersiveweapons.flare": "信号弹",
+	"entity.immersiveweapons.smoke_grenade": "烟雾弹",
+	"entity.immersiveweapons.smoke_grenade_arrow": "烟雾弹箭",
+	"entity.immersiveweapons.molotov_cocktail": "莫洛托夫鸡尾酒",
+	"entity.immersiveweapons.mortar_shell": "迫击炮弹",
+	"entity.immersiveweapons.burned_oak_boat": "烧焦的橡木船",
+	"entity.immersiveweapons.burned_oak_chest_boat": "烧焦的橡木箱船",
+	"entity.immersiveweapons.chair": "椅子",
+	"entity.immersiveweapons.mud_ball": "泥球",
+	"entity.immersiveweapons.lava_revenant": "熔岩复仇者",
+	"entity.immersiveweapons.rock_spider": "岩石蜘蛛",
+	"entity.immersiveweapons.celestial_tower": "天塔",
+	"effect.immersiveweapons.morphine": "吗啡",
+	"effect.immersiveweapons.bleeding": "流血",
+	"effect.immersiveweapons.alcohol": "醉酒",
+	"container.immersiveweapons.small_parts_table": "小零件工艺",
+	"container.immersiveweapons.tesla_synthesizer": "特斯拉聚合器",
+	"gui.jei.category.small_parts": "小零件工艺",
+	"gui.jei.category.tesla_synthesizer": "特斯拉聚合器",
+	"gui.jei.category.tesla_synthesizer.note": "*32倍的速度",
+	"immersiveweapons.subtitle.armor.tesla.equip": "特斯拉装甲叮当作响",
+	"immersiveweapons.subtitle.armor.tesla.effect": "特斯拉装甲轰击",
+	"immersiveweapons.subtitle.armor.tesla.power_down": "特斯拉装甲关闭",
+	"immersiveweapons.subtitle.armor.tesla.power_up": "特斯拉装甲开启",
+	"immersiveweapons.subtitle.armor.molten.equip": "熔融盔甲吱吱作响",
+	"immersiveweapons.subtitle.armor.ventus.equip": "温蒂斯盔甲咔擦作响",
+	"immersiveweapons.subtitle.armor.copper.equip": "铜装甲弯曲",
+	"immersiveweapons.subtitle.armor.cobalt.equip": "钴装甲叮当作响",
+	"immersiveweapons.subtitle.item.gun.flintlock_fire": "燧发枪射击",
+	"immersiveweapons.subtitle.item.gun.bullet_whizz": "子弹呼啸而过",
+	"immersiveweapons.subtitle.item.gun.flintlock_misfire": "枪打不出子弹了",
```

Comment:
```
The gun can't shoot any more
```
This is a sound effect, not a statement. Also, the gun can still shoot, just the particular attempt at firing failed because of poor ammunition. Should be something like "Gun misfires" or "Gun explodes".